test/unit/helpers/util/Expect.js

```diff
@@ -86,6 +86,10 @@ define([
 
   fn.fitp = runTest.bind({}, fit);
 
+  /**
+   * @deprecated This is very hacky rather solving the actual reason of flakiness.
+   *             Please slack eng-ui and think twice before you want to use this function.
+   */
```

Comment:
Best option for us is to work toward removing this method entirely. Can we create a follow-up JIRA to remove `tick` from these tests?

Reply:
There are 28 test files using `tick` and I'm not sure if a single JIRA would be useful or 28 tickets.

Reply:
😱 

Gotcha. I'm wondering if adding this deprecation notice is enough to encourage developers **not** to use this method. For now, we could probably get away with catching this during the review stage.

Reply:
I like the idea of deprecating this function, so devs don't use it anymore. But I think we should probably change the comment, since this is open source and eng-ui slack channel is only internal
Can we say something like
```
@deprecated This function is non deterministic and can affect the output of the tests
Instead use any of the Expect.wait* functions 
```

Reply:
@mauriciocastillosilva-okta @jmelberg-okta 
I hope it helps a little bit and we shall deny any code (new or updating exists) that still uses `tick`.
I'm all ears for any ideas that helps fixing flakyness. :)